pkg/backup/item_backupper.go

```diff
@@ -119,6 +119,7 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 	name := metadata.GetName()
 
 	log := logger.WithField("name", name)
+	log = log.WithField("resource", groupResource)
 	if namespace != "" {
```

Comment:
The one thing I noticed on the screenshot is that when we get to backing up the `PersistentVolume`, we still show a log field of `namespace=default`, which is incorrect since it's non-namespaced.  I think the easiest thing to do here is just remove the L123 `if` statement, and always execute L124.